{{cookiecutter.repostory_name}}/deploy.sh

```diff
@@ -2,10 +2,26 @@
 PRIMARY_IMAGE_HASH=$(docker-compose build app | tee /dev/tty \
     | sed '/^$/d' | grep "AS secondary-image" -B1 | head -1 | awk '{print $2}')
 docker-compose build  # build the rest of images
-docker-compose down -v --remove-orphans
-docker-compose up -d
+
+docker-compose stop app \
+{% if cookiecutter.use_celery == "y" -%}
+    celery-worker celery-beat \
+{%- endif %}
+{% if cookiecutter.use_flower == "y" -%}
+    celery-flower
+{%- endif %}
+
+docker-compose start app \
+{% if cookiecutter.use_celery == "y" -%}
+    celery-worker celery-beat \
+{%- endif %}
+{% if cookiecutter.use_flower == "y" -%}
+    celery-flower
+{%- endif %}
```

Comment:
is there some way in which we can avoid manually enumerating container names? skip the nginx and do everything else?

Reply:
So I resigned from jinja variables in favour of listing services using `docker-compose ps`